pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -83,7 +109,16 @@ class VmOverviewTabLibvirt extends React.Component {
     }
 
     close() {
-        this.setState({ showVcpuModal: false, showMemoryModal: false, showBootOrderModal: false });
+        this.setState({ showVcpuModal: false, showMemoryModal: false, showBootOrderModal: false, showFirmwareModal: false });
+    }
+
+    getOVMFBinariesOnHost(loaderElems) {
+        return Array.prototype.map.call(loaderElems, (loader, index) => {
+            const valueElem = loaderElems[index].getElementsByTagName('value');
```

Comment:
s/loaderElems[index]/loader/, and then you also don't need `index` any more.

Reply:
Fixed.